### ISSUE TITLE
Licenc Key

### ISSUE THREAD

**Issue by edcarlosm** [joaquinito01/Xtream-Codes-2.2.0-Nulled#6]
Amigo a licença esta invalida teria como gerar uma nova ou me mandar informação do seu docker

**Comment by wwhitty0** [comment]
Yes i have exact same issue guys. Be great to get a fix on this please?

**Comment by panchititin** [comment]
Yes i have exact same issue guys.  
PHP Warning:  file_get_contents(http://xtream-codes.com/softwares/iptv_panel/iptv_download.php?licence=3d5b0f8e468c332): failed to open stream: HTTP request failed! HTTP/1.1 500 Internal Server Er                ror
 in Xtream_Codes on line 301

**Comment by ahamris** [comment]
Still same error in may 2019

**Comment by CrimeNow** [comment]
same
